src/components/forms/DatePicker/DatePicker.tsx

```diff
@@ -29,6 +29,7 @@ type BaseDatePickerProps = {
   id: string
   name: string
   className?: string
+  validationStatus?: 'error' | 'success'
```

Comment:
In the PR I linked I made a `ValidationStatus` type that could be reused here as well

Reply:
If we merge yours first, I can update it in mine

Reply:
Please do, just pushed the test